godtools/App/Share/Data/LaunchCountRepository/LaunchCountCache.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  LaunchCountCache.swift
+//  godtools
+//
+//  Created by Levi Eggert on 8/16/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class LaunchCountCache {
+    
+    private let userDefaults: UserDefaults = UserDefaults.standard
+    
+    init() {
+        
+    }
+    
+    func getLaunchCount() -> AnyPublisher<Int, Never> {
+        
+        return userDefaults.publisher(for: \.launchCount)
+            .eraseToAnyPublisher()
+    }
+    
+    func getLaunchCountValue() -> Int {
+        
+        return userDefaults.launchCount
+    }
```

Comment:
Ahh shoot that makes sense that would happen.  That UseCase is initialized before notification applicationDidBecomeActive which increments the launch count.

Reply:
Gonna do some thinking on this one.

Reply:
I think I'll keep launch count just in case we want to do some other things with it.  Say maybe on fifth launch do something.

Reply:
Hey @rachaelblue, ended up creating that Singleton to hold onto those UseCase references and starting it after the launch count is incremented.

Reply:
yeah, we should keep launch count because there have been variations of requests to trigger things after # launches. so far we haven't had to directly use the count because there has been alternatives in firebase, but still would be good to have.

Reply:
Thanks @frett that's good to know!

Reply:
sounds good!